lib/BC/Util.cpp

```diff
@@ -381,10 +376,22 @@ llvm::GlobalVariable *FindGlobaVariable(llvm::Module *module,
 // Loads the semantics for the `arch`-specific machine, i.e. the machine of the
 // code that we want to lift.
 std::unique_ptr<llvm::Module> LoadArchSemantics(const Arch *arch) {
+    return LoadArchSemantics(arch, {});
+}
+
+std::unique_ptr<llvm::Module> LoadArchSemantics(const Arch *arch,
+                                                const std::vector<std::string> &sem_dirs) {
   auto arch_name = GetArchName(arch->arch_name);
-  auto path = FindSemanticsBitcodeFile(arch_name);
-  DLOG(INFO) << "Loading " << arch_name << " semantics from file " << path;
-  auto module = LoadModuleFromFile(arch->context, path);
+  // If `sem_dirs` does not contain the dir, fallback to compiled in paths.
+  auto path = FindSemanticsBitcodeFile(arch_name, sem_dirs, true);
+  // TODO(lukas): We can propagate error up, but we should first check each callsite
+  //              properly checks for possible error (this could not return pointer
+  //              without value before).
+  if (!path)
```

Comment:
`LOG_IF` or use `{` squigglies!

Reply:
Will use `{` as `LOG` macro will be removed next.